auto3dseg/algorithm_templates/swinunetr/scripts/train.py

```diff
@@ -579,7 +579,8 @@ def run(config_file: Optional[Union[str, Sequence[str]]] = None, **override):
 
                             if finished:
                                 break
-
+                        if not finished:
```

Comment:
`finished` might be an undefined variable if the loop at line 554 is empty, but I guess it's not a critical bug.